src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY_OPERATION = "buy";
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final int INDEX_OF_OPERATION = 0;
+    private static final int INDEX_OF_AMOUNT = 1;
+    private int amountOfBuy = 0;
+    private int amountOfSupply = 0;
```

Comment:
those should be local variables
![image](https://user-images.githubusercontent.com/81781479/195410727-d466ac54-8fe5-41cc-b97c-b836db159884.png)